daemon/src/db.rs

```diff
@@ -647,14 +644,7 @@ mod tests {
     }
 
     async fn setup_test_db() -> PoolConnection<Sqlite> {
-        let temp_db = tempdir().unwrap().into_path().join("tempdb");
-
-        // file has to exist in order to connect with sqlite
-        File::create(temp_db.clone()).unwrap();
-
-        let pool = SqlitePool::connect(format!("sqlite:{}", temp_db.display()).as_str())
-            .await
-            .unwrap();
+        let pool = SqlitePool::connect(":memory:").await.unwrap();
```

Comment:
Can we remove `tempfile` dependency now?

Reply:
We might be able to!